website/docs/api/docusaurus.config.js.md

```diff
@@ -2,6 +2,7 @@
 id: docusaurus.config.js
 title: docusaurus.config.js
 description: API reference for Docusaurus configuration file.
+slug: ./docusaurus.config.js
```

Comment:
./ means it's resolved against current api directory, so the url of this doc has changed and now has /api prefix

For now I think we should not change the urls of the docs when moving them